bluesky/callbacks/olog.py

```diff
@@ -158,7 +158,7 @@ def __init__(self, logbook):
         from pyOlog import SimpleOlogClient
         self.client = SimpleOlogClient()
         # Check at init time we are in an IPython session.
-        from IPython import get_ipython
+        from IPython import get_ipython  # noqa: F401
```

Comment:
We do this import twice. It feels it would be better to to stash `get_ipython` as an attribute of `self`, and reuse it later. Maybe too much for this PR.

Reply:
I see. But this is more of a code formatting PR. I guess we shouldn't change the code if it is not absolutely necessary.

Reply:
OK, let's keep it here, and it can be refactored/cleaned later.